### PR TITLE
[MWPW-134231] - PUF Block Unit Test

### PR DESCRIPTION
**Pull request by redvelvet511**
Added Integration test for PUF block

### CODE REVIEW

File: test/blocks/puf/puf.test.js [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+      puf.dispatchEvent(touchStartEvent);
+
+      const endTouch = new Touch({
+        identifier: Date.now(),
+        target: puf,
+        clientX: 0,
+        clientY: 50,
+      });
+
+      setTimeout(() => {
```
Comment (vhargrave) [inline test/blocks/puf/puf.test.js:102]: the problem with using timeouts in tests is that it makes the whole test execution a lot longer. 
It's much better to use fake timers - https://sinonjs.org/releases/latest/fake-timers/
If you want to see this in use, clone the milo project and just search for "clock"

File: test/blocks/puf/puf.test.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
```
Comment (vhargrave) [inline test/blocks/puf/puf.test.js:2]: nit: 2023

**Review by vhargrave — CHANGES_REQUESTED** [review]
Looking really good! The timeout thing should really be fixed, but asides from that it's just nit picking from me. 
Few additional notes: 
a change to real code to make things easier to test is usually not good practice (the change to instrument.js)
It seems like you've gained a little experience with the puf block. Would you feel comfortable spending an hour documenting it? https://main--express--adobecom.hlx.live/docs/authoring/ 
I have no idea what it is.

**Comment by redvelvet511** [comment]
Hi Victor! There are still some enhancements or changes being made in the PUF block so I'm going to re visit this when when it's done and also to refactor it including the fake timers.

**Comment by vhargrave** [comment]
@redvelvet511 can we close this PR and reopen when it's ready ? Just want to clean up the list of PRs needing a review :)

**Comment by redvelvet511** [comment]
> PR

Hi @vhargrave ! Yes, thank you. Will work on this when I'm back.

**Pull request closed without merging**